.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest] #, windows-latest]
+
+    name: Compile and Run Tests
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
What's the purpose of Mamba?

Reply:
it's the installer that installs the dependencies listed in the `environment.yml` file -- it's a cross-platform package manager.